scripts/org-setup.mjs

```diff
@@ -75,11 +75,6 @@ await $`sf force data bulk upsert --file ${bigTableCsv} --sobject BigTable__c --
 await $`sf force data bulk upsert --file ${bigTableCsv} --sobject BigTable__c --external-id Id --target-org jsforce-test-org`;
 await $`sf force data bulk upsert --file ${bigTableCsv} --sobject BigTable__c --external-id Id --target-org jsforce-test-org`;
 
-const pushTopicValues =
-  "Name='JSforceTestAccountUpdates' Query='SELECT Id, Name FROM Account' ApiVersion='54.0' NotifyForFields=Referenced NotifyForOperationCreate=true NotifyForOperationUpdate=true NotifyForOperationDelete=false NotifyForOperationUndelete=false";
-
-await $`sf data create record --sobject PushTopic --values ${pushTopicValues} --target-org jsforce-test-org`;
-
```

Comment:
This is used by the streaming e2e test, moved there so that the query looks for a specific account name.
This should help when tests are running on parallel, with the previous query it could get an `Account` record from other tests and now will just look for the account name that has an ID for the streaming test.

Reply:
happened here:
https://github.com/jsforce/jsforce/actions/runs/7617138418/job/20745503850?pr=1390#step:8:665